okio/src/main/kotlin/okio/internal/ByteString.kt

```diff
@@ -168,10 +168,42 @@ internal fun ByteString.commonEndsWith(suffix: ByteString) =
 internal fun ByteString.commonEndsWith(suffix: ByteArray) =
     rangeEquals(size - suffix.size, suffix, 0, suffix.size)
 
+internal fun ByteString.commonIndexOf(other: ByteString, fromIndex: Int): Int {
+  val limit = size - other.size
+  for (i in maxOf(fromIndex, 0)..limit) {
```

Comment:
should this be until and not `..` ?

Reply:
I almost always want `until` and not `..` because `..` is [inclusive, inclusive] which is rarely used in our code